packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -680,7 +684,7 @@ login:
     expires: never
     data_sensitivity:
       - interaction
-  third_party_auth_login_no_pw_view:
+  third_party_auth_no_pw_view:
```

Comment:
Confirming that this is intended to be in the `login` category, and not `third_party_auth`

Reply:
Thank you for calling that out.  It's a Third Party Auth event in the Google sheet.  I'll update.

@akkomar will that cause any issues?

Reply:
No, this will just be a new event from the platform point of view. Since its [current form](https://dictionary.telemetry.mozilla.org/apps/accounts_frontend/metrics/login_third_party_auth_login_no_pw_view) seems to be active only in stage ([monitoring graph from Glean dictionary](https://mozilla.cloud.looker.com/dashboards/1452?Event+Name=%22login.third_party_auth_login_no_pw_view%22&App+Name=Mozilla+Accounts+Frontend&Window+Start+Time=28+days&Channel=)) we won't be losing any useful data.

FWIW this already is under `login` category.